plugins/discovery-ec2/build.gradle

```diff
@@ -80,7 +102,11 @@ test {
 
   // this is needed to manipulate com.amazonaws.sdk.ec2MetadataServiceEndpointOverride system property
   // it is better rather disable security manager at all with `systemProperty 'tests.security.manager', 'false'`
-  systemProperty 'java.security.policy', "file://${buildDir}/tmp/java.policy"
+  if (BuildParams.inFipsJvm){
+    systemProperty 'java.security.policy', "=file://${buildDir}/tmp/java.policy"
```

Comment:
What does the prefixed `=` do here? I wasn't able to find documentation on that special syntax. We should add a comment explaining for future readers.

Reply:
I explain this here https://github.com/elastic/elasticsearch/blob/1fe526414adda0a3e12f395770f8b61cb5f8a4fa/buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy#L165, if you think we should copy the explanation in every use, happy to add it here too